docs/dev-guide/contributing/features.rst

```diff
@@ -38,27 +38,35 @@ Acceptance Criteria
   testing criteria, or other deliverables that need to be completed to define what "done"
   looks like.
 
-Once an author feels a story is ready, visit ``#pulp-dev`` and for a review of it. If
-the reviewer agrees the story is ready, the reviewer marks the ``Groomed`` flag as
-True. It is the author's responsibility to continue the conversation until the story is Groomed.
 
+Grooming
+--------
+
+Story grooming is the process of preparing a story to be accepted as a Sprint Candidate,
+ensuring that the Story meets the standards Outlined in the previous section.
```

Comment:
Nope. I actually went through and tried to remove Excessive Capitalization, so this is particularly funny.